tests/test_main.py

```diff
@@ -1 +1,88 @@
-# write your code here
+from app.main import get_human_age
+
+
+def test_output_type():
+    assert type(get_human_age(1, 1)) == list
+
+    is_int = True
+    if type(get_human_age(1, 1)) == list:
+        for i in get_human_age(30, 31):
+            if not type(i) == int:
+                is_int = False
+                break
+
+    assert is_int
+
+
+def test_zero_age():
+    assert get_human_age(0, 0) == [0, 0]
+
+
+def test_age_less_than_15():
+    expected = {
+        (2, 2): [0, 0],
+        (10, 10): [0, 0],
+        (14, 14): [0, 0]
+    }
+    values = [(2, 2), (10, 10), (14, 14)]
+
+    actual = {item[1]: get_human_age(values[item[0]][0], values[item[0]][1])
+              for item in enumerate(values)}
```

Comment:
You can include zero age here and rename test to `test_first_age` because you testing that ages less than 1.
Use `mark.parametrize` here and in other tests.